packages/react/src/components/Authenticator/FederatedSignIn/FederatedSignInButtons/FederatedSignInButton.tsx

```diff
@@ -28,5 +31,20 @@ export const FederatedSignInButton = (
     });
   };
 
-  return <Button onClick={handleClick}>{text}</Button>;
+  // TODO - get Google and Amazon icons
+  let iconComponent;
+  if (icon === 'facebook') {
+    iconComponent = <IconFacebook />;
+  }
```

Comment:
We can steal these SVGs rather than expecting `IconFacebook` to be exported as a primitive:

https://github.com/aws-amplify/amplify-js/blob/d815f8bb825163fcb5d6f713b235b1421e08b1a6/packages/amplify-ui-components/src/components/amplify-icon/icons.tsx#L31-L68

(I need to write a proposal/PR for our re-exporting of Material UI icons)

Reply:
Nice, sounds good.

Reply:
Removing this for now until we pull in the others.